boards/nucleo144-common/include/board_common.h

```diff
@@ -35,11 +35,18 @@ extern "C" {
  * @name    LED pin definitions and handlers
  * @{
  */
-#define LED0_PIN            GPIO_PIN(PORT_B, 0)
-#define LED0_MASK           (1 << 0)
-#define LED0_ON             (GPIOB->BSRR = LED0_MASK)
-#define LED0_OFF            (GPIOB->BSRR = (LED0_MASK << 16))
-#define LED0_TOGGLE         (GPIOB->ODR  ^= LED0_MASK)
+#if defined(STM32L496ZG)
+#define LED0_PORT           PORT_C
+#define LED0_PIN_NUM        7
+#else
+#define LED0_PORT           PORT_B
+#define LED0_PIN_NUM        0
+#endif
+#define LED0_PIN            GPIO_PIN(LED0_PORT, LED0_PIN_NUM)
+#define LED0_MASK           (1 << LED0_PIN_NUM)
+#define LED0_ON             (LED0_PORT->BSRR = LED0_MASK)
```

Comment:
this seems broken now: `LED0_PORT` is either `PORT_C` or `PORT_B`, where it should be translated to `GPIOC` and `GPIOB`, respectively.